File: 1672-richest-customer-wealth/1672-richest-customer-wealth.go
func maximumWealth(accounts [][]int) int {
    
    wealth := 0
    
    for i := 0; i < len(accounts); i++ {
        sum := 0
        for j := 0; j < len(accounts[0]); j++ {
            sum += accounts[i][j]
        }
        if sum > wealth {
            wealth = sum
        }
    }
    
    return wealth
}

File: 1672-richest-customer-wealth/1672-richest-customer-wealth.py
class Solution(object):
    def maximumWealth(self, accounts):
        """
        :type accounts: List[List[int]]
        :rtype: int
        """
        output = 0
        
        for i in range(len(accounts)):
            wealth = 0
            
            for j in range(len(accounts[0])):
                wealth = wealth + accounts[i][j]
                
            if wealth > output:
                output = wealth
        
        return output
        
            
        